packages/jumpstarter-driver-raspberrypi/jumpstarter_driver_raspberrypi/driver.py

```
@@ -1,63 +1,133 @@
+    raise ImportError("lgpio is not installed, lgpio might not be supported on your platform") from err
+
+from jumpstarter_driver_power.driver import PowerInterface
 
 from jumpstarter.driver import Driver, export
 
 
+class _GPIOBase(Driver):
+    """Base GPIO"""
+
+    pin: int
+    _h: int = field(init=False, repr=False)
```

Pylint (3.3.7)

[error] Invalid usage of field(), it should be used within a dataclass or the make_dataclass() function.

(E3701)